testing/bdd/features/timing/delay.feature

```diff
@@ -20,29 +20,29 @@ Feature: Delay of a document sequence
     # Negative value should really throw an exception
     | 00:00:10.0 | -00:00:00  | 00:00:10.0         |  
 
-#TODO: test that the order of documents in the input and output sequence is the same (FIFO) 
+#TODO: test that the order of documents in the input and output sequence is the same (FIFO)
 
 
   #TBC: this is a change to the spec (passive delay node)
   @skip
```

Comment:
So I can see several things here: 
- the two skipped scenarios. Should we leave them for #293 ?
- the TODO: we implement it for this PR ?
- the line about negative delay: we remove it ?